### PR TITLE
[PLATFORM-662] Show module warning notifications

### PR DESCRIPTION
**Pull request by timoxley**
Shows canvas notifications any time there's a "module warning" or "notification" event on the canvas main uiChannel.

To test: make a canvas something like this, start then stop:

image

I haven't yet figured out (or even looked very hard TBH) a way to trigger "notification" type events.

See https://streamr.atlassian.net/browse/PLATFORM-662 for more info. I added a new comment regarding the hopelessness of our situation.

### CODE REVIEW

File: app/src/editor/canvas/hooks/useCanvasNotifications.jsx [hunk 1]
```diff
@@ -65,6 +65,26 @@ export function pushErrorNotification(error) {
     })
 }
 
+export function pushWarningNotification(warning) {
+    console.warn(warning) // eslint-disable-line no-console
+    // eslint-disable-next-line react/no-danger
```
Comment (mondoreale) [inline app/src/editor/canvas/hooks/useCanvasNotifications.jsx]: Disabling [`react/no-danger`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-danger.md) shouldn't be necessary here.

File: app/src/editor/canvas/hooks/useCanvasNotifications.jsx [hunk 2]
```diff
@@ -65,6 +65,26 @@ export function pushErrorNotification(error) {
...
+    Notification.push({
+        title: warning.message || 'Warning',
+        icon: NotificationIcon.WARNING,
+        warning,
+    })
+}
+
+export function pushInfoNotification(info) {
+    console.info(info) // eslint-disable-line no-console
+    // eslint-disable-next-line react/no-danger
```
Comment (mondoreale) [inline app/src/editor/canvas/hooks/useCanvasNotifications.jsx]: And here.

Comment (timoxley) [reply app/src/editor/canvas/hooks/useCanvasNotifications.jsx]: Fixed 
97a001c

File: app/src/editor/canvas/hooks/useCanvasNotifications.jsx (hunk 1 above)
Comment (timoxley) [reply app/src/editor/canvas/hooks/useCanvasNotifications.jsx]: Fixed in 97a001c

**Comment by timoxley** [comment]
@tumppi good spotting, "fixed":

image

**Pull request merged**